pkgs/development/python-modules/shapely/1.8.nix

```diff
@@ -15,14 +15,15 @@
 }:
 
 buildPythonPackage rec {
-  pname = "Shapely";
+  pname = "shapely";
   version = "1.8.5";
   format = "pyproject";
```

Comment:
```suggestion
  pyproject = true;
```

Reply:
@Stunkymonkey , are you OK with adding suggested change to this PR or shell we merge this PR as is ?